src/js/components/Carousel/README.md

```diff
@@ -145,6 +145,16 @@ Defaults to
 <Next />
 ```
 
+**carousel.animationDuration**
```

Comment:
Can we make this `carols.animation.duration`? That will allow us to change other aspects of the animation within the same object.

Also, the default should be in `themes/base.js`.